docker/scripts/install_R_version

```diff
@@ -60,54 +60,62 @@ activate_R_${R_VERSION}
 
 function checkRInstallLog() {
   log_file=$1
-  if [ grep -E '(packages .* are not available)|(installation of package .* had non-zero exit status)' $log_file ]; then
+  if grep -E '(packages .* are not available)|(installation of package .* had non-zero exit status)' $log_file; then
     echo "Errors detected in R package installation."
     exit 1
   fi
 }
 
 # Install dependencies
 echo "Installing dependencies for R ${R_VERSION}"
-/usr/local/R/current/bin/R -e "chooseCRANmirror(graphics=FALSE, ind=58); \
+/usr/local/R/current/bin/R -e "chooseCRANmirror(graphics=FALSE, ind=1); \
```

Comment:
set mirror to "0-Cloud" which does "automatic redirection to servers worldwide, currently sponsored by Rstudio"

At the time of this change using mirror 58 I wasn't able to download some packages. But it is possible that it was just temporarily down.

Reply:
I like the fact, that 1 is 0 :-D